cidr-filtering-cni

```diff
@@ -7,12 +7,12 @@ logFile="${LOGFILE:-/var/log/cidr-filtering-cni.log}"
 
 exec 2>> $logFile
 
-NFT_TABLE_NETDEV=netdev
-NFT_TABLE_BRIDGE=bridge
+NFT_INGRESS_TABLE_TYPE=bridge
+NFT_EGRESS_TABLE_TYPE=bridge
```

Comment:
would it make sense to have a single variable for the family ? Both ingress and egress must use the same family ...

Reply:
Yes, Done